_sass/_features.scss

```diff
@@ -99,6 +99,31 @@ body.features-page {
 
 .feature {
     margin-bottom: $spacing-unit/2;
+
+    .feature-link {
+        margin-left: -$spacing-unit !important;
+    }
+}
+
+.feature, .feature-group {
+    .feature-anchor {
+        top: -($header-height + $header-border-top-height + $header-border-bottom-height + $spacing-unit/2);
+        display: block;
+        position: relative;
+    }
+
+    .feature-link {
+        display: none;
+    }
+
+    &:hover {
+        &> .feature-link {
```

Comment:
This rule is quite buggy:
![zrzut ekranu z 2016-09-15 20-13-12](https://cloud.githubusercontent.com/assets/3450050/18561628/0066ba2c-7b81-11e6-8421-74e8e90d9df8.png)

Here both `h2` and `h3` anchors have `hover` event triggered, and so both are shown.

I think the proper way to go is how this is implemented on GitHub (and it also seems more semantically correct for me):
```html
<h1><a id="user-content-intellij-rustgithubio" class="anchor" href="#intellij-rustgithubio" aria-hidden="true">...</a>intellij-rust.github.io</h1>
```
```css
.markdown-body .anchor {
    float: left;
    padding-right: 4px;
    margin-left: -20px;
    line-height: 1;
}
```